api/src/main/resources/liquibase-update-to-2.0.xml

```diff
@@ -9535,4 +9535,230 @@
         </sql>
     </changeSet>
 
+	<changeSet id="TRUNK-4730-20161114-1000" author="grindei" dbms="mysql">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="concept_class" columnName="date_changed"/>
```

Comment:
Since the fix version is 2.1, can you move these changesets to liquibase-update-to-2.1.xml